kayenta-wavefront/src/main/java/com/netflix/kayenta/wavefront/metrics/WavefrontMetricsService.java

```diff
@@ -0,0 +1,141 @@
+package com.netflix.kayenta.wavefront.metrics;
+
+import com.netflix.kayenta.canary.CanaryConfig;
+import com.netflix.kayenta.canary.CanaryMetricConfig;
+import com.netflix.kayenta.canary.CanaryScope;
+import com.netflix.kayenta.canary.providers.metrics.WavefrontCanaryMetricSetQueryConfig;
+import com.netflix.kayenta.wavefront.canary.WavefrontCanaryScope;
+import com.netflix.kayenta.wavefront.canary.WavefrontCanaryScopeFactory;
+import com.netflix.kayenta.wavefront.service.WavefrontTimeSeries;
+import com.netflix.kayenta.wavefront.security.WavefrontCredentials;
+import com.netflix.kayenta.wavefront.security.WavefrontNamedAccountCredentials;
+import com.netflix.kayenta.wavefront.service.WavefrontRemoteService;
+import com.netflix.kayenta.metrics.MetricSet;
+import com.netflix.kayenta.metrics.MetricsService;
+import com.netflix.kayenta.security.AccountCredentialsRepository;
+import com.netflix.spectator.api.Registry;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Singular;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.validation.constraints.NotNull;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+@Builder
+@Slf4j
+public class WavefrontMetricsService implements MetricsService {
+
+    @NotNull
+    @Singular
+    @Getter
+    private List<String> accountNames;
+
+    @Autowired
+    private final AccountCredentialsRepository accountCredentialsRepository;
+
+    @Autowired
+    private final Registry registry;
+
+    @Override
+    public String getType() {
+        return "wavefront";
```

Comment:
All the metric source modules where refactored to use static constants for the service type.
See: https://github.com/spinnaker/kayenta/blob/master/kayenta-stackdriver/src/main/java/com/netflix/kayenta/stackdriver/metrics/StackdriverMetricsService.java#L87

Reply:
👍 fixed